GS01SUMO/GS01SUMO_R00004_ingest.csv

```diff
@@ -41,8 +41,11 @@ mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data
 mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-12567/*.hex,GS01SUMO-RII11-02-CTDMOR019,recovered_inst,Available,
 mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo10/*.DAT,GS01SUMO-RII11-02-CTDMOR020,recovered_host,Available,
 mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13648/*.hex,GS01SUMO-RII11-02-CTDMOR020,recovered_inst,Available,
+#mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDBP_sn16-50137/SBE*.hex,GS01SUMO-RII11-02-DOSTAD031,recovered_inst,Not Expected,No data available
 mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdbp01/ctdbp01*.DAT,GS01SUMO-RII11-02-DOSTAD031,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDBP_sn16-50138/SBE*.hex,GS01SUMO-RII11-02-DOSTAD032,recovered_inst,Available,
 mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdbp02/ctdbp02*.DAT,GS01SUMO-RII11-02-DOSTAD032,recovered_host,Available,
+#mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDBP_sn16-50139/SBE*.hex,GS01SUMO-RII11-02-DOSTAD033,recovered_inst,Not Expected,No data available
 mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdbp03/ctdbp03*.DAT,GS01SUMO-RII11-02-DOSTAD033,recovered_host,Available,
 mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdbp01/ctdbp01*.DAT,GS01SUMO-RII11-02-FLORDG031,recovered_host,Available,

```

Comment:
This unit (3369) was lost at sea.

Reply:
Also phsen1 was lost at sea; phsen2 was damaged (hence no recovered_inst data).
And inductive pco2w1 and pco2w2 were lost at sea; pco2w3 was damaged (hence no recovered_inst data).

Reply:
Copy, good to know. I elected to not add those notes here because I don't like to use the ingest csvs to record those types of things (with the exception of gliders to note why there is not any recovered data). Let me know if you would prefer I enter those notes